app/routes/interestgroups/components/InterestGroupMemberList.js

```diff
@@ -40,7 +73,10 @@ const ListedUser = ({ user, role }: { user: User, role: string }) => (
     <Flex className={styles.row}>
       <ProfilePicture size={30} user={user} />
       <RoleIcon role={role} />
-      <Link to={`/users/${user.username}`}>{user.fullName}</Link>
+      <Link to={`/users/${user.username}`}>
+        <span className={roleStyle(role)}>{user.fullName} </span>
+        <span className={styles.suffix}>{suffix(role)}</span>
```

Comment:
This leaves a bunch of empty spans in the DOM, since most members don't have a role. You should instead check for a role before creating the element 😄